db/r2dec

```diff
@@ -0,0 +1,21 @@
+R2PM_BEGIN
+
+R2PM_GIT "https://github.com/wargio/r2dec-js"
+R2PM_DESC "[r2-r2pipe-node] an Experimental Decompiler"
+BINDIR="${R2PM_BINDIR}"
+
+R2PM_INSTALL() {
+	node --version || exit 1
+	npm install || exit 1
+	mkdir -p "${R2PM_BINDIR}" || exit 1
```

Comment:
this dir should be already mkdir'd by r2pm, but i guess its fine to have it here